src/bundler.ts

```diff
@@ -15,10 +22,30 @@ export async function isGemOutdated(): Promise<boolean> {
   return false;
 }
 
+export async function addGem(): Promise<void> {
+  await execInPath("bundle add ruby-lsp --group=development && bundle install");
```

Comment:
We can use `--require=false` here so it'd be added with `:require => false`.
(I don't like the old hash syntax but it's still better than nothing imo)